proto/meta.proto

```diff
@@ -60,6 +60,7 @@ message TableFragments {
     // Vnode mapping (which should be set in upstream dispatcher) of the fragment.
     common.ParallelUnitMapping vnode_mapping = 5;
     repeated uint32 state_table_ids = 6;
+    repeated uint32 upstream_fragment_ids = 7;
```

Comment:
How about also adapting a similar comment for `StreamActor.upstream_actor_id`? https://github.com/risingwavelabs/risingwave/blob/a635e4720e92842c1cc7c4065861b783ec4ee708/proto/stream_plan.proto#L446-L450